vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/InternalFieldBase.java

```diff
@@ -49,8 +51,8 @@
         extends AbstractSinglePropertyField<TComponent, TValue>
         implements CompositionNotifier, Focusable<TComponent>,
         HasAutocapitalize, HasAutocomplete, HasAutocorrect, HasClearButton,
-        HasClientValidation, HasHelper, HasLabel, HasPrefixAndSuffix, HasSize,
-        HasStyle, HasTooltip, HasValidation, HasValidator<TValue>,
+        HasClientValidation, HasHelper, HasLabel, HasPrefix, HasSize, HasStyle,
+        HasSuffix, HasTooltip, HasValidation, HasValidator<TValue>,
```

Comment:
Well, if you remove `HasPrefixAndSuffix` from here - you can completely delete it :D It's even worst now, because people are not going to get compile errors, there get nasty runtime errors and are wondering why `TextField instance HasPrefixAndSuffix` returns false instead of true :) Having `HasPrefixAndSuffix extends HasPrefix, HasSuffix` was meant to let the old interface in this component - thanks to the inheritance, the TextField should now be instance of all three interfaces.

Reply:
Thanks for clarification. I'll close this PR for now, we can get back to this in Vaadin 25.

Reply:
Thanks for listening :)  I should have said parts of the above in my original comment as well, you can't look in my head - that's why I wanted to remove the methods from the interface, so that Eclipse doesn't show them as deprecated, as they are moved from deprecated to the inherited new interfaces

Reply:
Got it, thanks. That makes a lot of sense 👍